src/app/lib/utilities/preprocessor/rules/cpsAssetPage/addBylineBlock/index.js

```diff
@@ -0,0 +1,35 @@
+import pathOr from 'ramda/src/pathOr';
+import deepClone from 'ramda/src/clone';
+import path from 'ramda/src/path';
+import convertToBylineBlock from './models';
+
+const getBylineBlock = json => {
+  const byline = pathOr(null, ['promo', 'byline'], json);
+
+  if (!byline) {
+    return json;
+  }
+
+  return byline;
+};
+
+const insertBylineBlock = originalJson => {
+  const json = deepClone(originalJson);
+  const type = path(['metadata', 'type'], json);
+
+  const blocks = path(['content', 'model', 'blocks'], json);
```

Comment:
We use `pathOr` in the method above... is there any reason why we are not doing the same here?

Reply:
We followed the style of the file `src/app/lib/utilities/preprocessor/rules/cpsAssetPage/addHeadlineBlock/index.js`, good spot.